### DISCUSSION TITLE
Time navigation buttons per panel

### DISCUSSION THREAD

**Discussion by massaquah** [grafana/grafana#60828] [Dashboards, Panels & Visualizations]
Hello,

I am trying to recreate the energy consumption bar chart from Home Assistant and I am particularly interested in the time selection / navigation bar in the upper right corner.

![Image 025](https://user-images.githubusercontent.com/6074180/209996624-12be5216-ab83-48a8-ad3a-1b9841854b32.png)

Having these quick selection buttons is much more comfortable than the default time selector in Grafana.
What makes it even more complicated is that I want to have these buttons per chart and not per dashboard.

Any ideas?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
